### ISSUE TITLE
Keyboard navigation (similar to Vimium)

### ISSUE THREAD

**Issue by ghost** [pop-os/cosmic-epoch#3642]
**Is your feature request related to a problem? Please describe.**
According to research mouse usage causes health issues [short term](https://www.tandfonline.com/doi/abs/10.1080/001401399184992) and [long term](https://pubmed.ncbi.nlm.nih.gov/21063306/). Ergonomic mouses help, but don't prevent the repetitive strain inherent to mice (https://pmc.ncbi.nlm.nih.gov/articles/PMC9265546/). Keyboard navigation is better and should be encouraged. Examples of such implementations: https://chromewebstore.google.com/detail/vimium-c-all-by-keyboard/hfjbmagddngcpeloejdejnfgbamkjaeg, https://qutebrowser.org/ and https://zed.dev/docs/vim. 

It would be nice if Cosmic offers Vimium-C like keyboard navigation to improve not only the health, but also productivity for people. Since such a design would require less moving of the hands to do actions like clicking a link. 

One of the issues of just using an extension like Vimium-C is trust. Since you give it full permisions to your browser data, so a built-in solution systemwide  would be more secure.

**Describe the solution you'd like**
A opt-in toggle to enable navigating through the system similar to how Vimium C allows navigating through sites.

**Describe alternatives you've considered**
Using tab and shift tab but that is not as fast or nice to use. 

**Additional context**
This would work in all apps. Example implementation: https://github.com/AlfredoSequeida/hints
